ci/kokoro/refdocs.sh

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+# For docuploader to work
+export LC_ALL=C.UTF-8
+export LANG=C.UTF-8
+
+if [[ -z "${CREDENTIALS:-}" ]]; then
+  CREDENTIALS=${KOKORO_KEYSTORE_DIR}/73713_docuploader_service_account
+fi
+
+STAGING_BUCKET="docs-staging"
+
+BUILD_DIR="cmake-out"
+
+# work from the git root directory
+cd "$(dirname "$0")/../../"
+
+# Try to determine the branch
+if [[ -z "${KOKORO_GITHUB_COMMIT:-}" ]]; then
+  echo "KOKORO_GITHUB_COMMIT not found, aborting..."
+  exit 1
+else
+  BRANCH=`git branch --no-color --contains ${KOKORO_GITHUB_COMMIT} \
```

Comment:
Here and elsewhere, you should use `$(...)` instead of backticks. Also, enclose `${KOKORO_GITHUB_COMMIT}` in quotes. The build will fail for any shellcheck warnings, you can run `ci/check-style.sh` inside Docker locally.

Reply:
Done